### DISCUSSION TITLE
Chat community

### DISCUSSION THREAD

**Discussion by ghost** [gavv/httpexpect#135] [Ideas]
Hello @gavv , can you create a chat room at https://gitter.im ? I think it will be nice to have a community near you, it's easy just register and press Add Room, from repository, select the gavv/httpexpect, give some permissions to the app and you're ready, I have to ask you somethings also :) 

add these to your readme after create the room: 

``` markdown

[![Chat Widget]][Chat]

[Chat Widget]: https://img.shields.io/badge/community-chat-00BCD4.svg?style=flat-square
[Chat]: https://gitter.im/gavv/httpexpect
```

**Comment by gavv** [comment]
It already exists :) See first line in README.

**Comment by ghost** [comment]
Lawl, I searched for it but I didn't found it.. my eyes are broken, sorry then :)
